.babelrc

```diff
@@ -1,7 +1,18 @@
 {
-  "presets": ["latest", "stage-2"],
-  "plugins": [
-    "add-module-exports",
-    "transform-react-jsx",
+  "presets": [
+    [
+      "env",
+      {
+        "targets": {
+          "browsers": [
+            ">0.25%",
+            "not ie 11",
+            "not op_mini all"
+          ]
+        }
+      }
+    ],
+    "react",
+    "stage-2"
```

Comment:
what do you still need stage 2 for?

Reply:
I think this broke when running tests because of a spread or something, I can look into removing that though it might not actually be necessary

Reply:
oh, its `static propTypes = {`. 🤔

Reply:
ahh yeah, static fields, had the same issue with something else last night due to babel config